nms/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -194,75 +194,124 @@ const StyledMenu = withStyles({
   />
 ));
 
+export type SubscriberActionType = 'add' | 'edit' | 'delete';
```

Comment:
Use enum and object.freeze
https://stackoverflow.com/questions/287903/how-can-i-guarantee-that-my-enums-definition-doesnt-change-in-javascript